doc/api/next_api_changes/behavior/28177-REC.rst

```diff
@@ -0,0 +1,14 @@
+(Sub)Figure.get_figure
+~~~~~~~~~~~~~~~~~~~~~~
+
+...in future will by default return the direct parent figure, which may be a SubFigure.
+This will make the default behavior consistent with the
+`~matplotlib.artist.Artist.get_figure` method of other artists.  To control the
+behavior, use the *root* parameter.
+
+
+(Sub)Figure.set_figure
+~~~~~~~~~~~~~~~~~~~~~~
+
+...in future will always raise an exception.  The parent and root figures of a
```

Comment:
Is there a reason to announce this, but not formally (pending?) deprecate this?

Reply:
Oh, you mean this should be in the deprecations section?